test/ETO/ETOCommitment.js

```diff
@@ -1224,7 +1282,6 @@ contract("ETOCommitment", ([deployer, admin, company, nominee, ...investors]) =>
           MAX_TICKET_EUR_ULPS: Q18.mul(15000000),
           ENABLE_TRANSFERS_ON_SUCCESS: false,
           PUBLIC_DISCOUNT_FRAC: publicDiscount,
-          ALLOW_RETAIL_INVESTORS: true,
```

Comment:
where is ETOConstraints with non-transferable token explicitly passed?

Reply:
does this play a role in that specific test? I now added the command for that here but I don't think it's used, no?

Reply:
hmmm nooo, you are right, I think I just wanted to have small ticket sizes. OK